qa/rpc-tests/feature_fedpeg.py

```diff
@@ -77,6 +85,7 @@ def setup_network(self, split=False):
         self.fedpeg_script = "512103dff4923d778550cc13ce0d887d737553b4b58f4e8e886507fc39f5e447b2186451ae"
         parent_chain_signblockscript = '51'
         for n in range(2):
+            used_cookie_auth = n==1
             rpc_u, rpc_p = rpc_auth_pair(n)
```

Comment:
nit: maybe move these to lines to right before the `if used_cookie_auth:` line because they are only used in that block afaics.